test/acceptance/tests/basic/basic_test.go

```diff
@@ -29,120 +30,131 @@ func TestValidation_RetryJoinRequired(t *testing.T) {
 }
 
 func TestBasic(t *testing.T) {
-	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
-		TerraformDir: "./terraform/basic-install",
-		Vars:         suite.Config().TFVars(),
-		NoColor:      true,
-	})
-	defer func() {
-		if suite.Config().NoCleanupOnFailure {
-			logger.Log(t, "skipping resource cleanup because -no-cleanup-on-failure=true")
-		} else {
-			terraform.Destroy(t, terraformOptions)
-		}
-	}()
-	terraform.InitAndApply(t, terraformOptions)
+	cases := []bool{false, true}
 
-	// Wait for consul server to be up.
-	var consulServerTaskARN string
-	retry.RunWith(&retry.Timer{Timeout: 3 * time.Minute, Wait: 10 * time.Second}, t, func(r *retry.R) {
-		taskListOut, err := shell.RunCommandAndGetOutputE(t, shell.Command{
-			Command: "aws",
-			Args: []string{
-				"ecs",
-				"list-tasks",
-				"--region",
-				suite.Config().Region,
-				"--cluster",
-				suite.Config().ECSClusterARN,
-				"--family",
-				fmt.Sprintf("consul_server_%s", suite.Config().Suffix),
-			},
-		})
-		r.Check(err)
+	for _, secure := range cases {
+		t.Run(fmt.Sprintf("secure: %t", secure), func(t *testing.T) {
+			randomSuffix := strings.ToLower(random.UniqueId())
```

Comment:
Changing it to creating a random suffix for each test because otherwise we can't re-run tests locally for the same ECS cluster from `setup-terraform`. That is because AWS doesn't delete secrets permanently; it just schedules them for deletion. So if you re-run any test that has tls enabled, your second run will fail because AWS will think you're trying to create a secret that's scheduled for deletion.